.github/workflows/publish.yml

```
@@ -0,0 +1,54 @@
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+name: Deploy Extension
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.0.2
+      - name: Set Job Env
+        if: github.event_name == 'release' && github.event.action == 'published'
+        run: .github/actions/setJobEnv.sh
+      - name: Setup Node
+        uses: actions/setup-node@v3.5.0
```

```suggestion
        uses: actions/setup-node@v3
```

Same comment as above